tests/integration/generated/test_complete_run_chrysalis.cfg

```diff
@@ -116,6 +116,7 @@ years = "1850:1854:2", "1850:1854:4",
   ref_years = "1850-1851",
   reference_data_path = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/atm/180x360_aave/clim"
   run_type = "model_vs_model"
+  sets = "lat_lon","lat_lon_land","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere",
```

Comment:
@forsyth2 hi, I'm skimming at this PR to try figure out the problem. It sounds like only one e3sm_diags viewer is created for including both lat_lon and lat_lon_land? I had a [comment](https://github.com/E3SM-Project/zppy/pull/534#discussion_r1419679983) earlier. We should have a separate task i.e. [[ lnd_monthly_180x360_aave_mvm ]] for the "lat_lon_land" set, because it is more straightforward, for instance this set depends on the the land data paths. (in the implementation here, I don't think the land data are used)

Reply:
Oh, at the time, I thought that advice was just for model-vs-obs, not model-vs-model since that seemed to be working. So yes, that's a good idea here too. I'll separate them and see if anything changes in the plots. Thanks.